src/watch.ts

```diff
@@ -101,22 +123,22 @@ export function useWatch({
 					watch.pollRate = pollRate || DEFAULT_POLL_RATE_SECS
```

Comment:
It's not really part of the scope of this PR, but I noticed that `DEFAULT_POLL_RATE_SECS` is already assigned as default value to the `pollRate` parameter (line 47).

```suggestion
					watch.pollRate = pollRate
```

Reply:
I'm not sure I agree. If a poll rate of zero is specified, it'll default back sensibly. I think it's ok as it is.

Reply:
Good point, I didn't think about that.
What about removing the assignment at line 47 instead though?

Also what would happen if a negative pollRate is given? Isn't this addressed by haystack-nclient in some way?
I feel like `pollRate > 0 ? pollRate : DEFAULT_POLL_RATE_SECS` would be a more explicit approach (although less optimized).

Reply:
Yes I agree for checking the negative numbers. I can make that change.

I think the default argument is different from checking whether it's valid or not. I still think the default argument makes sense.

Reply:
@riccardolederj2 I've updated the code so it deals with a negative pollRate value.